apps/web-giddh/src/app/all-modules/all-modules.component.ts

```diff
@@ -1,15 +1,86 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ComponentFactoryResolver, ViewChild } from '@angular/core';
+import { ManageGroupsAccountsComponent } from '../shared/header/components';
+import { ElementViewContainerRef } from '../shared/helpers/directives/elementViewChild/element.viewchild.directive';
+import { ModalDirective } from 'ngx-bootstrap';
+import { take } from 'rxjs/operators';
+import { Store, select } from '@ngrx/store';
+import { AppState } from '../store';
+import { GeneralActions } from '../actions/general/general.actions';
+import { GroupWithAccountsAction } from '../actions/groupwithaccounts.actions';
+import { GeneralService } from '../services/general.service';
 
 @Component({
     selector: 'all-modules',
     templateUrl: './all-modules.component.html',
     styleUrls: ['all-modules.component.scss']
 })
+
 export class AllModulesComponent implements OnInit {
 
-    constructor() {
+    @ViewChild('addmanage') public addmanage: ElementViewContainerRef;
+    @ViewChild('manageGroupsAccountsModal') public manageGroupsAccountsModal: ModalDirective;
+
+    public activeCompany: any;
+
+    constructor(private componentFactoryResolver: ComponentFactoryResolver, private store: Store<AppState>, private generalActions: GeneralActions, private groupWithAccountsAction: GroupWithAccountsAction, private generalService: GeneralService) {
```

Comment:
Consider breaking the injected dependencies in multiple lines for better readability
Import `vatSupportedCountries` variable as it is referenced in HTML